pkg/apiserver/apiserver.go

```diff
@@ -112,6 +116,38 @@ const (
 	MaxTimeoutSecs = 600
 )
 
+func (g *APIGroupVersion) GetAPIPrefix() string {
```

Comment:
GetAPIPrefix and GetAPIGroup will not be needed if #14156 gets in. And we have utility function to split group/version in pkg/api/util/group_version.go